packages/menu/src/MenuItem.tsx

```diff
@@ -219,18 +225,22 @@ const MenuItem = React.forwardRef(
           tabIndex={disabled ? -1 : undefined}
         >
           <div
-            className={cx(titleTextStyle, {
-              [activetitleTextStyle]: active,
-              [disbaledTextStyle]: disabled,
-            })}
+            className={cx(
+              titleTextStyle,
+              {
+                [activeTitleTextStyle]: active,
+                [disabledTextStyle]: disabled,
+              },
+              titleTextClassName,
+            )}
           >
             {children}
           </div>
           {description && (
             <div
               className={cx(descriptionTextStyle, {
                 [activeDescriptionTextStyle]: active,
-                [disbaledTextStyle]: disabled,
+                [disabledTextStyle]: disabled,
```

Comment:
I'd coordinate with @brookescarlett around changes to this file in particular. I know she also has a pull request in where this is fixed.

Reply:
This will be addressed as part of removing MenuItem as a dependency for SideNavItem. I'll revert the changes I made beyond pulling out the repeated types.